trace_viewer/core/find_control.html

```diff
@@ -112,7 +112,6 @@
 
     filterFocus: function(e) {
       this.controller.reset();
-      this.filterTextChanged();
```

Comment:
I removed this because when a user inputs an unaccepted ui state string, an overlay is invoked and focus is no longer in the find box. Once the user dismisses the overlay and focuses back on the find box, the overlay is incorrectly invoked again because of this function even though the text was exactly the same as before.

I'm a little unsure about this removal -- as far as I can tell this shouldn't be there, and I couldn't find anything that broke after I removed it..